packages/http-server/src/server.ts

```diff
@@ -97,67 +110,80 @@ export const createServer = (operations: IHttpOperation[], opts: IPrismHttpServe
         inputOutputValidationErrors.forEach(validation => {
           const message = `Violation: ${validation.location.join('.') || ''} ${validation.message}`;
           if (validation.severity === DiagnosticSeverity[DiagnosticSeverity.Error]) {
-            request.log.error({ name: 'VALIDATOR' }, message);
+            components.logger.error({ name: 'VALIDATOR' }, message);
```

Comment:
Nice, this is probably way better than our current code _anyway_

Reply:
👍